### ISSUE TITLE
Frequency functions for collections

### ISSUE THREAD

**Issue by JonathanStarup** [flix/flix#7007]
For example `def frequency(l: List[t]): Map[t, Int32] = ???`

**Comment by magnus-madsen** [comment]
PRs are welcome :) You can find information on dev setup in the docs folder.

**Comment by LionelMeli** [comment]
Hi Jonathan, I don't know how I can close this ticket (PR is merge with master).
Maybe you should assign me to this ticket in order I close it.

**Comment by magnus-madsen** [comment]
I think we should leave the ticket open and discuss whether to add more frequency functions for other collections.

**Comment by LionelMeli** [comment]
Sure. No problem.

**Comment by magnus-madsen** [comment]
I don't think we need more frequency functions in the standard library. We can have them as packages.
